plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtStartCommandWrapper.java

```diff
@@ -116,7 +116,7 @@ public Answer execute(final StartCommand command, final LibvirtComputingResource
                     }
                 }
                 // try to patch and SSH into the systemvm for up to 5 minutes
-                for (int count = 0; count < 10; count++) {
+                for (int count = 0; count < 30; count++) {
```

Comment:
should the comment above be updated to read "up to 15 minutes"?

Reply:
@rhtyd is this change part of this commit ?